### PR TITLE
Skatkov am lab 1

### PR DESCRIPTION
**Pull request by craftmaster1231**
Look like it is ready

### CODE REVIEW

File: SkatkovAM/lab1.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#include <iostream>
+#include <limits>
+using namespace std;
+float a,b;
```
Comment (Toxa-man) [inline SkatkovAM/lab1.cpp]: Give meaningful names to variables

File: SkatkovAM/lab1.cpp [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+int onOffInd,inputInd=1; //ind:0=right input; 1= wrong input; on
+char operation;
+void hello(){
+    cout<<"Hello. im calculator ^_^"<<endl;
+}
+
+void onOffCheck(){
+    if(a==0 && b==0 && operation=='0'){
+        onOffInd=1;
```
Comment (Toxa-man) [inline SkatkovAM/lab1.cpp]: Functions should communicate with each other through return values, not global variables. If you decided to use function - consider rewriting them and eliminating all global variables

**Review by Toxa-man — COMMENTED** [review]
Code issues.
Please create separate folder for each lab work

File: SkatkovAM/lab1.cpp (hunk 2 above)
Comment (craftmaster1231) [reply SkatkovAM/lab1.cpp:13]: Is it about all functions, or only  simple like that one?

**Pull request closed without merging**